src/constants.js

```diff
@@ -196,14 +197,17 @@ export const DefaultValues = {
   [SettingIds.MUTE_INVISIBLE_PLAYER]: false,
   [SettingIds.SCROLL_VOLUME_CONTROL]: false,
   [SettingIds.BLACKLIST_KEYWORDS]: {},
-  [SettingIds.HIGHLIGHT_KEYWORDS]: {
-    0: {
-      id: 0,
-      type: Types.MESSAGE,
-      status: null,
-      keyword: defaultHighlightKeywords(),
-    },
-  },
+  [SettingIds.HIGHLIGHT_KEYWORDS]:
+    twitch.getCurrentUser() != null
+      ? {
+          0: {
+            id: 0,
+            type: Types.MESSAGE,
+            status: null,
+            keyword: twitch.getCurrentUser().name,
```

Comment:
if a user is not logged in, isn't `getCurrentUser()` returning `undefined`? That would result in `Uncaught TypeError: Cannot read property 'name' of undefined`

Reply:
The ternary before would catch that `getCurrentUser()` != `null`